tests/test_cli.py

```diff
@@ -0,0 +1,4 @@
+def test_argparser():
+    sys.argv = "covert enc --recipient asdf -r asdf -Arrp recipient1".split()
+    a = argparser()
+    assert a.recipients == ['recipient1', 'recipient2']
```

Comment:
The red symbol at the end means that this file does not end with a newline. I suggest you configure your editor to always add a newline at the end. It is good to also configure it to remove any whitespace at the end of each line (and on empty lines), although there are no such issues in this PR.

Reply:
Apparently the red symbol is not rendered with this comment, just in case you were wondering, but it can be seen in git diff or in the full diff that Github shows.